src/components/general/RequestWorkflow/models.ts

```diff
@@ -17,17 +19,22 @@ export type WorkflowPerson = {
     mail: string;
     name?: string;
     phoneNumber?: string;
-}
+};
+
+export type WorfklowPendingPerson = {
+    person: WorkflowPerson;
+    stepId: string;
+};
 
 export type WorkflowStep = {
     id: string;
-    state: 'Pending' | 'Active' | 'Approved' | 'Skipped';
+    state: WorkflowStepState;
     name: string;
     description: string;
-    started: Date | null; 
-    completed: Date | null; 
+    started: Date | null;
```

Comment:
How about we populate the WorkflowStep with a pendingPerson parameter instead of having pendingPerson in a separate prop? We will then have to manipulate in workflow object that we get from the api in the app. Something like
`workflow.steps.map(s => s.id === "proposal" ? {...s, pendingPerson: "azureUniqueID"} : s)`

Reply:
Good idea!